src/spawn.c

```diff
@@ -67,6 +67,7 @@
 #else
 # include "support.h"
 #endif
+#include "utils.h"
```

Comment:
No no of course I don't want to change that function, sorry I was unclear.  By "it" I wanted to refer to the code path here, not the function itself, so yeah I meant the code here should rather use `g_locale_to_utf8()` instead.